src/pages/[platform]/build-a-backend/storage/copy/index.mdx

```diff
@@ -64,24 +88,43 @@ const copyFile = async () => {
     const response = await copy({
       source: {
         key: 'srcKey',
-        accessLevel: 'private' // optional 'guest', 'private', or 'protected'. Defaults to 'guest'.
+        accessLevel: 'protected' // optional 'guest', 'private', or 'protected'. Defaults to 'guest'.
         targetIdentityId: 'targetIdentityId' // optional, set it to other user's identity ID if copy from other user
       },
       destination: {
         key: 'destKey',
-        accessLevel: 'private' // optional 'guest', 'private', or 'protected'. Defaults to 'guest'.
+        accessLevel: 'protected' // optional 'guest', 'private', or 'protected'. Defaults to 'guest'.
       },
     });
   } catch (error) {
     console.error('Error', err);
   }
 };
 ```
+</Block>
+</BlockSwitcher>
 
 ## Copy files within the same access levels
 
-You can copy a file from the specified key to another key within the same [File Access Level](/[platform]/build-a-backend/storage/configure-access/) (Defaults to 'guest').
+You can copy a file from the specified path to another path within the same [File Access Level](/[platform]/build-a-backend/storage/configure-access/).
```

Comment:
yeah i thought about this a bit. So the plan for docs is to keep this docs as CLI support and not update to gen2. So in that case since the setup and everything here remains gen1 i thought it might make sense to give the usecase in `path` usage. Or atleast thats the idea